src/patternfly/components/DualListSelector/dual-list-selector.scss

```diff
@@ -128,26 +166,31 @@
 .pf-c-dual-list-selector__list {
   display: flex;
   flex-direction: column;
-}
 
-.pf-c-dual-list-selector__list-item {
-  background-color: var(--pf-c-dual-list-selector__list-item--BackgroundColor);
+  .pf-c-dual-list-selector__list {
+    --pf-c-dual-list-selector__item--PaddingLeft: var(--pf-c-dual-list-selector__item--list__item--PaddingLeft);
 
-  &:hover {
-    --pf-c-dual-list-selector__list-item--BackgroundColor: var(--pf-c-dual-list-selector__list-item--hover--BackgroundColor);
-  }
+    .pf-c-dual-list-selector__list-item.pf-m-expandable {
+      --pf-c-dual-list-selector__item--PaddingLeft: var(--pf-c-dual-list-selector__item--list__item--m-expandable--PaddingLeft);
+    }
 
-  &:focus-within {
-    --pf-c-dual-list-selector__list-item--BackgroundColor: var(--pf-c-dual-list-selector__list-item--focus-within--BackgroundColor);
+    .pf-c-dual-list-selector__list {
+      --pf-c-dual-list-selector__item--PaddingLeft: var(--pf-c-dual-list-selector__item--list--list__item--PaddingLeft);
+    }
   }
+}
+
+.pf-c-dual-list-selector__list-item {
 
-  &.pf-m-selected {
-    --pf-c-dual-list-selector__list-item--BackgroundColor: var(--pf-c-dual-list-selector__list-item--m-selected--BackgroundColor);
-    --pf-c-dual-list-selector__item-text--Color: var(--pf-c-dual-list-selector__list-item--m-selected__item-text--Color);
+  // stylelint-disable
+  &.pf-m-expanded {
+    .pf-c-dual-list-selector__item > .pf-c-dual-list-selector__item-main > .pf-c-dual-list-selector__item-toggle > .pf-c-dual-list-selector__item-toggle-icon > i {
```

Comment:
If this is to keep nested icons from rotating, you can just use this.

Note I'm styling `.pf-c-dual-list-selector__item-toggle-icon` instead of `i`, since that wouldn't rotate the icon in react. That also will require `display: inline-block;` on `__item-toggle-icon` so that it will transform, and an initial var for `--pf-c-dual-list-selector__item-toggle-icon--Transform: 0;`

```
&.pf-m-expanded {
  > .pf-c-dual-list-selector__item .pf-c-dual-list-selector__item-toggle-icon {
    --pf-c-dual-list-selector__item-toggle-icon--Transform: rotate(--pf-c-dual-list-selector__list-item--m-expanded__item-toggle-icon--Rotate);
  }
}
```